### DISCUSSION TITLE
Is there a way to print a dependency tree?

### DISCUSSION THREAD

**Discussion by agzam** [raxod502/straight.el#1156] [Q&A]
I use straight.el indirectly through my Doom config. I am not exactly sure which parts of package management Doom relies on straight.el for.

I have noticed that it sometimes pulls transient packages, e.g., evil-collection may pull in annalist.el. I often have trouble figuring out which package pulls what. Is there a way to print the dependency tree to understand better what was pulled and installed including the dependencies?

**Comment by progfolio** [comment]
See the `straight-dependencies` and `straight-dependents` commands.
The latter should prompt for a package and print a list of packages which depend on it.

Do those commands help?
